dbt_coverage/__init__.py

```diff
@@ -507,14 +512,17 @@ def load_files(project_dir: Path) -> Catalog:
         catalog_table = catalog.get_table(table_name)
         manifest_source_table = manifest.sources.get(table_name, {})
         manifest_model_table = manifest.models.get(table_name, {})
+        manifest_seed_table = manifest.seeds.get(table_name, {})
         manifest_table_tests = manifest.tests.get(table_name, {})
 
         for catalog_column in catalog_table.columns.values():
             manifest_source_column = manifest_source_table.get(catalog_column.name)
             manifest_model_column = manifest_model_table.get(catalog_column.name)
+            manifest_seed_column = manifest_seed_table.get(catalog_column.name)
             manifest_column_tests = manifest_table_tests.get(catalog_column.name)
 
-            doc = (manifest_source_column or manifest_model_column or {}).get('description')
+            doc = (manifest_source_column or manifest_model_column or manifest_seed_column or {}) \
```

Comment:
@sweco my personal preference would be to create a separate variable, like `picked_manifest_column` or something like that, but that's not all that important in here 🙂

Reply:
That's a fair feedback, thanks. Will do! 😊